cmd/clusterctl/cmd/describe_cluster_test.go

```diff
@@ -261,31 +264,44 @@ func Test_TreePrefix(t *testing.T) {
 				return obectjTree
 			}(),
 			expectPrefix: []string{
-				"Object/root",
-				"├─Object/child1",
-				"│ │           ├─C1.1", // first condition child gets pipes, children pipe and ├─
-				"│ │           └─C1.2", // last condition child gets pipes, children pipe and └─
-				"│ └─Object/child1.1",
-				"└─Object/child2",
-				"  │           ├─C2.1", // first condition child gets spaces, children pipe and ├─
-				"  │           └─C2.2", // last condition child gets spaces, children pipe and └─
-				"  └─Object/child2.1",
+				"Object/root                           ",
+				"├─Object/child1                       ",
+				"│ │           ├─C1.1  True      0s    ", // first condition child gets pipes, children pipe and ├─
+				"│ │           └─C1.2  True      0s    ", // last condition child gets pipes, children pipe and └─
+				"│ └─Object/child1.1                   ",
+				"└─Object/child2                       ",
+				"  │           ├─C2.1  True      0s    ", // first condition child gets spaces, children pipe and ├─
+				"  │           └─C2.2  True      0s    ", // last condition child gets spaces, children pipe and └─
+				"  └─Object/child2.1                   ",
```

Comment:
The test looks like it is supposed to test on the prefixes of each of the row. The test data should not need to include any of the other columns or padding. If you switch the matcher function to test using `strings.HasPrefix` ([here](https://github.com/kubernetes-sigs/cluster-api/pull/5942/files#diff-3ffcf7422cc24118431d2182585eb57ef4af304db60c5e3279b6d5ec01fde497R364)) I think you can solve the problem. 
The test cases will become a lot simpler and easy to maintain and reason about.

Reply:
done